caddy/pmtiles_proxy.go

```diff
@@ -70,8 +73,12 @@ func (m Middleware) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddy
 	for k, v := range headers {
 		w.Header().Set(k, v)
 	}
-	w.WriteHeader(statusCode)
-	w.Write(body)
+	if statusCode == 200 {
+		http.ServeContent(w, r, "", time.UnixMilli(0), bytes.NewReader(body))
+	} else {
+		w.WriteHeader(statusCode)
+		w.Write(body)
+	}
```

Comment:
I thought caddy might handle this automatically but it doesn't look like it...

@bdon Since this is identical to main.go for `pmtiles serve` do you think it's worth moving this into server.go? Or just have clients do it manually?

Reply:
yeah I think we should centralize it into `server.go` instead of repeating it in the CLI entry point  + caddy module.